bgpd/bgp_debug.c

```diff
@@ -1029,8 +1029,8 @@ DEFUN (debug_bgp_bestpath_prefix,
        "IPv4 prefix\n"
        "IPv6 prefix\n")
 {
-	struct prefix argv_p;
 	int idx_ipv4_ipv6_prefixlen = 3;
+	struct prefix argv_p = {};
```

Comment:
I hate to ask this but would it be better to actually check the return value of str2prefix?

Reply:
not that str2prefix could ever really fail here

Reply:
@donaldsharp how about `assert()`? Maybe convert to `DEFPY` and call the mask apply? If this is just dragging problem around we might need to drop this and refactor the functions to properly handle CLI input...

Reply:
I wouldn't argue here at all for that approach.

Reply:
the DEFPY approach.  Then we will know it will always be right